packages/kit/src/runtime/client/client.js

```diff
@@ -673,10 +673,23 @@ export function create_client({ target, base }) {
 				}
 			});
 
-			if (import.meta.env.DEV) {
+			if (__SVELTEKIT_DEV__) {
```

Comment:
btw as a drive-by I replaced the `import.meta.DEV` occurences withour own global